BambooTracker/midi/RtMidi/RtMidi.pri

```diff
@@ -5,6 +5,10 @@ HEADERS += \
     $$PWD/RtMidi.hpp
 
 linux {
+    contains(DEFINES, __UNIX_JACK__) {
+        DEFINES += JACK_HAS_PORT_RENAME
```

Comment:
I believe this only works if we limit what JACK version we support to JACK2. This should pose a problem on macOS where the commonly used JACK version from Homebrew is actually JACK1.

If we really can't detect the JACK version with qmake in the same way RtMidi's build systems do via header inspection or some other way, then maybe we should start thinking about making custom defines for convenience. `-D_USE_JACK1`, `-D_USE_JACK2`, `-D_USE_PULSE` etc.

Reply:
I find that [`CONFIG` variable and qmake condition used it](https://doc.qt.io/qt-5/qmake-language.html#configuration-and-scopes).
How about making the `CONFIG` options `use_pulse`, `use_jack1` and `use_jack2` and add each `DEFINES` in these condition?

Reply:
That looks like a better idea, yeah.